scrooge-generator/src/main/resources/javagen/finagleServiceFunction.java

```diff
@@ -3,17 +3,18 @@
     try {
```

Comment:
Why convert from twitter future to CompletableFuture inside this apply method? At the end it just returns twitter Future